### DISCUSSION TITLE
What Is The Default Behaviour of OMR 6?

### DISCUSSION THREAD

**Discussion by chubx** [Ysurac/openmptcprouter#3438] [Q&A]
Hi, I'm new to this OMR project, and I have lots of questions that I wasn't able  to find answers  via search engine. So, I'm trying my luck here.

**Background**:

- I have a two WAN configuration, 1 Starlink, 1 Fiber.
- Starlink configured as WAN1 via VLAN5, and set as **Master**
- Fibre configured as WAN2 via VLAN6, and set as **Enabled**
- I'm using a VLAN switch
- So far the configurations are working

**My Tests**:

- I am testing the config whether I can use the router for VoIP calls or Google Meet or Zoom meeting without interruptions, when one of the WAN goes down.
- To test or simulate the scenario, I am simply do a ping test via ping `ping 1.1.1.1` command, then physically disconnect WAN1, the Starlink. The result: ping stops
- However, when I do the same, but on the WAN2 interface, ping continues -- and to me, that means no interruptions after disconnecting WAN2

**Questions**:

- Can anyone shed some light here please?
- Is that how OMR 6 works?
- I was expecting Bonding with Redundancy, is this even achievable using OMR 6?
- I have tried setting the WAN2 as backup,  it works, but how about a Bonding configuration with active backup/failover?

**Comment by Ysurac** [comment]
Ping use the VPN, in any case it's not yet perfect with the VPN, it's working better for TCP that use the Proxy.
Some VPN may work better in some case, you can choose them in Wizard.
